.github/workflows/cleanup.yml

```diff
@@ -65,4 +65,32 @@ jobs:
               --name "$account" \
               --resource-group "$RESOURCE_GROUP" \
               --yes
+          done
+
+  cleanup-akv:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Log into Azure
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.MANAGED_ID_CLIENT_ID }}
```

Comment:
let's use the best practice to document how to get these values. In this case you can refer to the doc in docs describing these.

Reply:
Can this be a separate PR? There's several places which uses this to login some of which are already on main. 

I'm also pro having the code self document to avoid desyncing between docs and code, and I don't know how better I can describe where to get the client id than `MANAGED_ID_CLIENT_ID` other than just converting that name into prose e.g. "_Go to the managed identity and use it's client id_"